src/app/scripts/cac/control/cac-control-directions-list.js

```diff
@@ -28,7 +28,8 @@ CAC.Control.DirectionsList = (function (_, $, ShareModal, MapTemplates) {
             googlePlusShareButton: '.modal-list-google',
             shareModalButton: '.share-directions',
             stepByStepClass: 'body-step-by-step',
-            sidebarBannerClass: 'body-has-sidebar-banner'
+            hasSidebarBannerClass: 'body-has-sidebar-banner',
```

Comment:
A nit to pick: I would prefer this keep its name as `sidebarBannerClass`, in keeping with the naming of other selectors. Not clear what the intent of the rename is here.